my_list.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(list)
+    @list = list
+  end
+
+  def each
+    @list.each { |e| yield(e) if block_given? }
+  end
+end
+
+list = MyList.new([1, 2, 3, 4])
```

Comment:
- [ ] Please, make changes to use `list = MyList.new(1, 2, 3, 4)` in place an array according the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/create_your_own_enumerable.md#:~:text=irb%3E%20list%20%3D%20MyList.new(1%2C%202%2C%203%2C%204)). 🐱

Reply:
Ups, didn't realize that! 
Its fixed now!